src/tests/functional/shared_test_classes/src/single_op/paged_attention.cpp

```diff
@@ -0,0 +1,154 @@
+   auto block_indices_begins = ov::op::v0::Constant::create(ov::element::i32, {i32_attr.block_indices_begins.size()}, i32_attr.block_indices_begins);
+
+    /*
+    * [9]: scale, optional
+    * [10]: sliding_window, optional
+    * [11]: alibi_slopes, optional
+    * [12]: max_context_len
+    * shape: [], type: i32
+    */
+    auto scale_const = ov::op::v0::Constant::create(ov::element::f32, {}, {misc_attrs.scale.value()});
+    auto sliding_window_const = ov::op::v0::Constant::create(ov::element::i32, {}, {misc_attrs.sliding_window.value()});
+    auto alibi_slopes_const = ov::op::v0::Constant::create(ov::element::f32, {misc_attrs.alibi_slopes.size()}, misc_attrs.alibi_slopes);
+    auto max_context_len_const = ov::op::v0::Constant::create(ov::element::i32, {}, {misc_attrs.max_context_len});
+
+
+    std::shared_ptr<ov::Node> paged_attn;
+    if (!rotation_attr.has_value()) {
+        paged_attn = std::make_shared<ov::op::PagedAttentionExtension>(
+            params[0],
+            params[1],
+            params[2],
+            params[3],
+            params[4],
+            past_lens,
+            subsequence_begins,
+            block_indices,
+            block_indices_begins,
+            scale_const,
+            sliding_window_const,
+            alibi_slopes_const,
+            max_context_len_const);
+    } else {
+    /*
+    * [13]: rotated_block_indices​, optional​
+    * shape: [num_rotated_blocks]​, type: i32
+    * [14]: rotation_deltas​, optional​
+    * shape: [num_rotated_blocks, BLOCK_SIZE]​ || [num_rotated_blocks, 1]​, type: i32
+    * [15]: rotation_trig_lut​, optional​
+    * shape: [max_num_batched_tokens / BLOCK_SIZE, head_size]​ || [max_num_batched_tokens, head_size], type: f16
+    */
+    auto rotated_block_indices = ov::op::v0::Constant::create(
+        ov::element::i32,
+        {rotation_attr.value().rotated_block_indices.size()},
+        rotation_attr.value().rotated_block_indices);
+    auto rotation_deltas = ov::op::v0::Constant::create(
+        ov::element::i32,
+        rotation_attr.value().rotation_deltas,
+        std::vector<int>{0});
+    auto rotation_trig_lut = ov::op::v0::Constant::create(
+            ov::element::f32,
+            rotation_attr.value().rotation_trig_lut,
+            std::vector<float>{0});
+
+    paged_attn = std::make_shared<ov::op::PagedAttentionExtension>(
+        params[0],
+        params[1],
+        params[2],
+        params[3],
+        params[4],
+        past_lens,
+        subsequence_begins,
+        block_indices,
+        block_indices_begins,
+        scale_const,
+        sliding_window_const,
+        alibi_slopes_const,
+        max_context_len_const,
+        rotated_block_indices,
+        rotation_deltas,
+        rotation_trig_lut);
+    }
+    ov::ResultVector results{std::make_shared<ov::op::v0::Result>(paged_attn->output(0))};
```

Comment:
FYI:
Create result is not required the output of `paged_attn` can be provided to create model and result will be added by model

Reply:
Done